pkg/webui/console/store/middleware/logics/gateways.js

```diff
@@ -100,16 +112,33 @@ const startGatewayStatisticsLogic = createLogic({
   cancelType: [
     gateways.STOP_GTW_STATS,
     gateways.UPDATE_GTW_STATS_FAILURE,
-    gateways.UPDATE_GTW_STATS_UNAVAILABLE,
   ],
   warnTimeout: 0,
-  validate ({ getState, action }, allow, reject) {
+  processOptions: {
+    dispatchMultiple: true,
+  },
+  transform ({ action }, next) {
+    const { meta = {}} = action
+
+    let transformed = action
+    if (!meta.timeout) {
+      transformed = { ...action, meta: { ...meta, timeout: 5000 }}
```

Comment:
What's the reasoning behind decorating this here?
(I know this is only refactored, but I'm just wondering this)

Reply:
to make sure the default polling timeout is set

Reply:
Seems like a very verbose approach as opposed to just assuming this value as a default in the `process` hook, but I guess I am missing something here.

Reply:
you are right. it can be simplified